.eslintrc.json

```diff
@@ -321,9 +349,162 @@
             {
                 "allowAllPropertiesOnSameLine": true
             }
-        ]
+        ],
+        "@typescript-eslint/ban-ts-comment": "off",
```

Comment:
💭 Judging by your use of descriptions on most of your annotations, maybe it'd be nice to set this to allow-with-description?

Reply:
Updated to use `ts-expect-error`.